src/main/java/com/racing/domain/Cars.java

```diff
@@ -0,0 +1,76 @@
+package com.racing.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Cars {
+    List<Car> carList;
+
+    public Cars() {
+        carList = new ArrayList<>();
+    }
+
+    public Cars(List<Car> carList) {
+        this.carList = carList;
+    }
+
+    public void addCars(Car car) {
+        carList.add(car);
+    }
+
+    public List<Car> getCarList() {
+        return this.carList;
+    }
```

Comment:
```suggestion
    public List<Car> getCarList() {
        return Collections.unmodifiableList(this.carList);
    }
```

읽기 전용 데이터로 변경해서 내보내셔야 외부의 영향을 받지 않게 됩니다.
`getCarList().add(car)` 를 수행하면 `Cars`가 변경되어 버립니다.